contributor/05-benefits-of-contribution.md

```diff
@@ -0,0 +1,113 @@
+# Draft comments
+
+Outline for this segment
+
+What are the benefits of contributing - both from an individual as well as from
+a business perspective. 
+
+# Article
+
+## Benefits of becoming an InnerSource Contributor
+
+Contributors are the life blood of Inner Source projects: Every project that is
+run as an Inner Source project comes both with the promise and with the ultimate
+goal of expanding their development team beyond the original founders, tapping
+into the potential for further collaborators amongs users (also sometimes
+referred to as customers in corporations) of that project.
+
+However what would motivate an individual developer to spend time on a project
+that is not under the direction of their manager? What would motivate a manager
+to make time for their developers to improve projects that are not 100% under
+their purview?
+
+### Individual motivation
+
+The most obvious motivation is what typically draws early contributors into open
+source as well:
+
+Rememember that annoying bug you've been working around for so long? The time
+and energy mainainting those workarounds costs? What if instead of waiting for
+the upstream team to fix that issue sometime in the future, you could go ahead
+and fix it yourself? Often in a scratch your own itch kind of approach first
+time contributors start with fixing issues in projects they depend on for their
+daily work to reduce the number of workarounds in their own codebase.
+
+When deciding whether to create and contribute a fix instead of maintaining your
+own workaround - think about the benefit that the contribution will bring to
+your own changes: Instead of working in isolation, those working on the upstream
+project will be able to not only review but also improve your solution. You get
+support and mentoring which greatly speeds up your own development effort.
+
+Spending more time with others means that over time you will learn how the team
+works, how it organises itself, which tooling it uses in which way to build
+their project. Oftentimes your own projects wil benefit from that experience:
+Instead of only reading about some new library or build system you'll be able to
+gain practical experience with it before going ahead and introducing that in
+your own projects. Working on more than one core project means that you will be
+exposed to a larger ecosystem from which to draw best practices and solutions to
+challenges.
+
+A nice side effect of being able to spend some your time in other teams means
+that your reputation and impact expands the boundaries of your own team. So in
+addition to yourself learning and growing from others you get to influence
+projects, both in a very specific way by your own contributions but also by
+sharing your experience and knowledge on project tooling and setup that might
+help the upstream project improve and accelerate development cycles.
+
+### Team level motivation
```

Comment:
One point that is in the [outline](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/outline.md) that I don't seen here is _ Do the right thing for the company ..._.  Is there a way to include it?